Tests/SwiftUITests/ViewExtensionTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  EraseToAnyViewTests.swift
+//  SwifterSwift
+//
+//  Created by Jevon Mao on 6/18/21.
+//  Copyright © 2021 SwifterSwift. All rights reserved.
+//
+
+@testable import SwifterSwift
+import XCTest
+import SwiftUI
```

Comment:
You need to wrap this in `#if canImport`, just like the implementation. Look at the other test files to see an example.